ckanext/datapusher/templates/package/resource_read.html

```diff
@@ -0,0 +1,12 @@
+{% ckan_extends %}
+
+{% block action_manage %}
```

Comment:
try inserting into the front of `action_manage_inner` so that it appears in the same order as the tabs when editing.

Reply:
add similar links in `package/snippets/resource_item.html` at the front of `resource_item_explore_inner` and `package/snippets/resources.html` at the front of `resources_list_edit_dropdown_inner` so that this tab appears along with the others for faster navigation.

These were recently added as part of #7590

Reply:
@wardi cool! The conditional action link has been added to these template snippets in datapusher ext now. And `action_manage` updated to `action_manage_inner`